### DISCUSSION TITLE
Geolocation of iModels

### DISCUSSION THREAD

**Discussion by AfxentiouNikolas** [iTwin/community#50] [Q&A]
Greetings,

I am facing an issue with the geolocation; due to the new version, all my buildings are slightly shifted to the left. Does anyone have any ideas on how I can fix this using code? I tried making changes in ViewSetup.ts (located in the common folder with the map layer option), but it didn't work.

Kind regards,
Nikolas

![Screenshot 2024-08-27 125405](https://github.com/user-attachments/assets/7b5dfc69-0d24-4189-9404-6f79531d7ae4)
![Screenshot 2024-08-27 125412](https://github.com/user-attachments/assets/13430780-d3dd-4860-a563-1538bb54a8d2)

**Comment by GytisCepk** [comment]
> I am facing an issue with the geolocation; due to the new version, all my buildings are slightly shifted to the left.

Could you provide the version you are currently using, and the one you used previously ? Also, are you working on web or desktop application ?

> Does anyone have any ideas on how I can fix this using code?

Before attempting to address this programmatically, I recommend verifying that source files are geolocated correctly.
